gunicorn/reloader.py

```diff
@@ -105,7 +105,7 @@ def run(self):
             self._dirs = self.get_dirs()
 
             for dirname in self._dirs:
-                self._watcher.add_watch(dirname, mask=self.event_mask)
+                self._watcher.__load_trees(dirname)
```

Comment:
Don't we need `self.event_mask` any more?

Reply:
Yes - this slipped past - I will push a commit